save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/controller/AgentsController.kt

```diff
@@ -81,7 +83,8 @@ class AgentsController(
                 .flatMap { agentIds ->
                     agentService.saveAgentsWithInitialStatuses(
                         agentIds.map { id ->
-                            Agent(id, execution)
+                            val containerName = dockerClient.inspectContainerCmd(id).exec().name
```

Comment:
Let's hide this logic in `DockerAgentRunner` via `DockerService`, because `DockerClient` is missing in Kuberenetes and orchestrator will simply fail to start

Reply:
But in this case we will need to change signature of methods, and return containerName from `createContainerFromImage()`, for example, that's what you mean?

Reply:
Yes, something like this